contents/blog/how-and-why-track-your-website-with-posthog.md

```diff
@@ -0,0 +1,68 @@
+---
+date: 2022-09-10
+title: 'How (and why) to track your website with PostHog'
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ['Guides']
+author: ["joe-martin"]
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+---
+
+PostHog may have been built for product analytics, but that doesn’t mean you can only deploy it on your core product — you can also use it to gather analytics from your marketing website too. Internally at PostHog this is exactly what we do, tracking analytics from both the product and this very website within a single project. 
+
+Using PostHog to gather analytics in this way isn’t just an excuse for us to ‘dogfood’ our own product. It’s also a best practice that we strongly recommend to other organizations for several reasons…
+
+> Already want to track your marketing website with PostHog? Find out [how to install our JavaScript snippet](/docs/integrate/client/js) in just a few minutes!
+
+### 1. Get better data
+<BorderWrapper>
+    <Quote
+        imageSource="/images/customers/filip.png"
+        size="md"
+        name="Filip Stanev"
+        title="Co-founder, Saga"
+        quote={`“PostHog tracks more events than Mixpanel. For one event we defined, Mixpanel reported 6,343 occurrences last month. PostHog reported what we know to be the true figure, 8,169 — almost 30% more than Mixpanel.”`}
+    />
+</BorderWrapper>
+
+Web analytics tools, such as Google Analytics, are all well and good — but if your tracking stops as soon as a user logs in to your platform then can you really track their entire user journey? 
+
+Deploying PostHog across both your website and product enables you not only to track their entire journey, but also to unlock important insights into how your marketing site is performing. You can compare pageviews to purchases, for example, to understand your conversion rate with a [funnel insight](/manual/funnels) — and you can use a [path insight](/manual/paths) to visualize that journey completely.
+
+Ultimately, PostHog can do almost everything that Google Analytics can do and more. And, speaking of Google Analytics…
+
+### 2. Simplify your stack (and get rid of Google Analytics)
+
+Reducing your stack can have a few overlooked benefits, including greater data consistency. No two analytics tools are the same and each can be impacted by adblockers differently — so if you’re using Google Analytics on your website and PostHog on your product then it can be tough to get comparable results. This is especially true if you’re self-hosting PostHog, as this enables you to use first-party cookies and [gather up to 50% more data than other analytics tools](/customers/saga).
```

Comment:
Had we?

Reply:
```suggestion
Reducing your stack can have a few overlooked benefits, including greater data consistency. No two analytics tools are the same and each can be impacted by adblockers differently — so if you’re using Google Analytics on your website and PostHog on your product then it can be tough to get comparable results. This is especially true if you’re self-hosting PostHog -- which users such as Saga do in order to [gather up to 50% more data than other analytics tools](/customers/saga).
```

Reply:
I'll defer to @timgl on this, but pretty sure this is something that came up around the pricing page.